tests/test_azureplugin.py

```diff
@@ -307,7 +307,9 @@ def _get_proper_extensions_response(upper = False):
         region = region.upper()
```

Comment:
if we are not going to use f-string and the upper check, (as lines 310-312) in favour of more explicit info, shouldn't we remove this ? or keep this and remove lines 310-312 ?

Reply:
Sorry, I do not understand. You added the part that triggers on `upper`, I just changed it such that we do not use format string which is not available on Python 3.4 (SLE 12). If the `upper` part id not used why did you add it in the first place?

Reply:
so before, I added f-string as:
```python
region = 'foo'
if upper:
    region = region.upper()
bar = f"{foo}"
```
and AFAICT the intention is to use upper but not f-string
```python
if upper:
    data += 'USEAST'
else:
    data += 'useast'
```
instead of having two locations in data (for the purpose of the test) and removing the now unnecesary
```
if upper:
    region = region.upper()
```
so, I meant to remove the previous `if-upper+region.upper` and adapt the `data +=useast` to be lower or upper case accordingly